client/src/config/config.js

```diff
@@ -0,0 +1,4 @@
+
+
+
+export const backEndApi = "https://cyf-youtube-api.onrender.com";
```

Comment:
suggestion for you use .env to hide your important links 
https://create-react-app.dev/docs/adding-custom-environment-variables/